database-plugins/src/main/java/io/cdap/plugin/db/common/DBBaseConfig.java

```diff
@@ -68,10 +70,19 @@
   @Description("The existing connection to use.")
   private DBConnectorConfig connection;
 
+  @Name(JDBC_PLUGIN_TYPE)
+  @Description("Type of the JDBC plugin to use. This is the value of the 'type' key defined in the JSON file " +
+    "for the JDBC plugin. Defaults to 'jdbc'.")
+  @Nullable
+  public String jdbcPluginType;
+
   public boolean getEnableAutoCommit() {
     return enableAutoCommit == null ? false : enableAutoCommit;
   }
 
+  public String getJdbcPluginType() {
+    return jdbcPluginType == null ? DBUtils.PLUGIN_TYPE_JDBC : jdbcPluginType;
+  }
```

Comment:
nit: add a newline between this method and the next

Reply:
done :)